internal/compress/lz4/lz4.go

```diff
@@ -0,0 +1,56 @@
+package lz4
+
+import (
+	"io"
+
+	lz4 "github.com/pierrec/lz4/v3"
+)
+
+// Header is type alias of lz4.Header.
+type Header = lz4.Header
+
+// Reader represents an interface for Reader of lz4.
+type Reader interface {
+	io.Reader
+}
+
+// Writer represents an interface for Writer of lz4.
+type Writer interface {
+	io.WriteCloser
+	Header() *Header
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
Is it really needed?

I think it's better that lz4Io have a NewWriterLevel function like gzip.